src/main/java/org/datadog/jmxfetch/Filter.java

```diff
@@ -49,20 +52,15 @@ public String toString() {
         // ### OR
         // object: singleValue
         // ###
-        try {
-            return (ArrayList<String>) toCast;
-        } catch (ClassCastException e) {
-            return new ArrayList<String>() {
-                {
-                    add(((String) toCast));
-                }
-            };
+        if (toCast instanceof List) {
+            return (List<String>) toCast;
         }
+        return Collections.singletonList((String) toCast);
```

Comment:
I just want to make sure the result of this call is not expected to be modified anywhere. I see one assignment to a instance variable `this.excludeTags`

Reply:
You're right, there is some add operations on it afterward